docs/product/security/ai-ml-privacy.mdx

```diff
@@ -0,0 +1,49 @@
+---
+title: "AI/ML: Protecting User Privacy"
+sidebar_order: 10
+description: "Learn about Sentry's approach to AI/ML privacy"
+---
+
+Throughout Sentry’s history, we’ve operated under a policy of [privacy by default](https://sentry.io/lp/privacy-by-default/). This same principal applies to our work in the Artificial Intelligence (AI) and Machine Learning (ML) space, where we want to be just as [transparent](https://blog.sentry.io/terms-of-service-update/) about what data we’re using and why.
+
+Sentry is at a juncture where data and experience quality is much harder to maintain using traditional heuristics-based approaches. For example, fingerprinting error events as part of creating groups, has gotten a lot more complicated with the rise of JavaScript, and the use of extensions and third-party services. To train and validate models for grouping, notifications, and issue priority, Sentry will now need access to additional service data in order to be able to deliver a better user experience.
+
+## General Consent
+
+In view of the new data we need to collet to improve Sentry’s ability to train internal models (to better fingerprint errors and prioritize data within the Sentry system), we've added a new “Service Data Consent” agreement. You can find it by going to the **Settings** page in [Sentry](https://sentry.io) and clicking on the "Legal & Compliance" link under the "Usage & Billing" section.
```

Comment:
```suggestion
In view of the new data we need to collet to improve Sentry’s ability to train internal models (to better fingerprint errors and prioritize data within the Sentry system), we've added a new “Service Data Consent” agreement. You can find it by going to the **Settings** page in **Sentry** and clicking on the ["Legal & Compliance"](https://sentry.sentry.io/orgredirect/organizations/:orgslug/settings/legal/) link under the "Usage & Billing" section.
```
Just learned how to add a deep link that will take users to this specific location using their org slug :)